incubator/haproxy-ingress/templates/NOTES.txt

```diff
@@ -3,16 +3,8 @@ The haproxy-ingress controller has been installed.
 {{- if contains "NodePort" .Values.controller.service.type }}
 Get the application URL by running these commands:
 
-{{- if (not (empty .Values.controller.service.nodePorts.http)) }}
-  export HTTP_NODE_PORT={{ .Values.controller.service.nodePorts.http }}
-{{- else }}
-  export HTTP_NODE_PORT=$(kubectl --namespace {{ .Release.Namespace }} get services -o jsonpath="{.spec.ports[0].nodePort}" {{ template "haproxy-ingress.fullname" . }})
-{{- end }}
-{{- if (not (empty .Values.controller.service.nodePorts.https)) }}
-  export HTTPS_NODE_PORT={{ .Values.controller.service.nodePorts.https }}
-{{- else }}
-  export HTTPS_NODE_PORT=$(kubectl --namespace {{ .Release.Namespace }} get services -o jsonpath="{.spec.ports[1].nodePort}" {{ template "haproxy-ingress.fullname" . }})
-{{- end }}
+  export HTTP_NODE_PORT=$(kubectl --namespace {{ .Release.Namespace }} get services -o jsonpath="{.spec.ports[0].nodePort}" {{ template "haproxy-ingress.fullname" . }}-controller)
```

Comment:
Should we also add  `{{- if (not (empty .Values.controller.service.nodePorts.http)) }}`  these condition line

Reply:
Change to  `{{- if (not (empty .Values.controller.service.nodePorts)) }}`

Reply:
This do works. But I have a question.
From my perspective `controller.service.nodePorts` never declared in values.yaml, and that means `controller.service.nodePorts` = nil and nil = empty.
`controller.service.httpsPorts[0].nodePort=(insert number here)` and `controller.service.httpPorts[0].nodePort=(insert number here)` is the one responsible for setting Node Port with `--set`.
So since `controller.service.nodePorts` will never be set or declared, why insert those check in there? The logic will always return `false`. This is just a curious thought.